js/viz/docs/docBaseWidget.js

```diff
@@ -34,9 +34,10 @@ var BaseWidget = {
 
     /**
     * @pseudo CommonVizPalette
-    * @type Array<string>|string
-    * @default "Default"
-    * @acceptValues "Default" | "Soft Pastel" | "Harmony Light" | "Pastel" | "Bright" | "Soft" | "Ocean" | "Vintage" | "Violet" | "Carmine" | "Dark Moon" | "Soft Blue" | "Dark Violet" | "Green Mist"
+    * @type Array<string>|Enums.VizPalette
+    * @default "Material"
+    * @deprecatedAcceptValues "Default"
```

Comment:
Remove \@deprecatedAcceptValues "Default". We will mark it as deprecated in the "Enums.VizPalette" enum in a future PR.